devicetypes/plaidsystems/spruce-valve.src/spruce-valve.groovy

```diff
@@ -0,0 +1,76 @@
+/**
+ *  Copyright Plaid Systems 2020
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ *
+
+11-2020
+ * valveDuration slider capability added back to presentation
+ * tabs and trim whitespace
+
+**/
+
+metadata {
+	definition (name: "Spruce Valve", namespace: "plaidsystems", author: "Plaid Systems", mnmn: "SmartThingsCommunity", vid: "558e73d3-c800-3669-b276-1d4352eda12b"){
+		capability "Actuator"
+		capability "Valve"
+		capability "Sensor"
+		capability "Health Check"
+		capability "heartreturn55003.valveDuration"
+
+		command "open"
+		command "close"
```

Comment:
`open` and `close` are part of the `Valve` capability, and so these are redundant.

Reply:
Just bumping up Steven's comment.

Reply:
@natec007 Please change the code according to Steven's comment.